src/views/UserEdit/UserEdit.js

```diff
@@ -83,6 +84,7 @@ class UserEdit extends React.Component {
         defaultDeliveryAddressTypeId: null,
       },
       username: '',
+      type: USER_TYPES.PATRON,
```

Comment:
are you suggesting `userType.PATRON`?

Reply:
are you suggesting `userType.PATRON`?

Reply:
No, I'm suggesting this attribute should be named `userType` instead of `type` because the form-id is `userType` and the constant is named `USER_TYPES` and the field label is `User type`. You will make every developer's life easier if you use the same name for the same thing.